kyo-combinators/shared/src/main/scala/kyo/Combinators.scala

```diff
@@ -227,8 +229,11 @@ extension [A, S](effect: A < S)
       * @return
       *   A computation that produces the result of this computation with Abort[Throwable] effect
       */
-    def explicitThrowable(using Flat[A], Frame): A < (S & Abort[Throwable]) =
-        Abort.catching[Throwable](effect)
+    def resurrect(using Flat[A], Frame): A < (S & Abort[Throwable]) =
+        Abort.run[Throwable](effect: A < (S & Abort[Throwable])).map:
+            case Result.Success(v)                      => v
+            case err: Result.Fail[Throwable] @unchecked => Abort.get(err)
+            case err: Result.Panic                      => Abort.fail(err.exception)
```

Comment:
It seems this could be: 
```suggestion
            case Result.Success(v) => v
            case Result.Error(ex)     => Abort.fail(ex)
```